servicetalk-client-api/build.gradle

```
@@ -30,7 +30,9 @@ dependencies {
   testImplementation testFixtures(project(":servicetalk-concurrent-internal"))
   testImplementation project(":servicetalk-concurrent-test-internal")
   testImplementation project(":servicetalk-test-resources")
-  testImplementation "junit:junit:$junitVersion"
+  testImplementation "org.junit.jupiter:junit-jupiter-api:$junit5Version"
+  testImplementation "org.junit.jupiter:junit-jupiter-params:$junit5Version"
+  testRuntimeOnly "org.junit.jupiter:junit-jupiter-engine:$junit5Version"
   testImplementation "org.mockito:mockito-core:$mockitoCoreVersion"
   testImplementation "org.hamcrest:hamcrest-library:$hamcrestVersion"
```

For future PRs, please place `testRuntimeOnly` dependency after all `testImplementation` dependencies. We try to sort by the scope first:

```suggestion
  testImplementation "org.mockito:mockito-core:$mockitoCoreVersion"
  testImplementation "org.hamcrest:hamcrest-library:$hamcrestVersion"

  testRuntimeOnly "org.junit.jupiter:junit-jupiter-engine:$junit5Version"
```